src/containers/ArtistData/ArtistData.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
+import Artist from '../../components/Artist/Artist';
+import Paging from '../../components/Paging/Paging';
+import { fetchReleases } from '../../services/musicbrainz/musicbrainz';
+
+const ArtistData = () => {
+  const [releases, setReleases] = useState([]);
+  const [page, setPage] = useState(0);
+  let { artistName, artistId, pageNum } = useParams();
+  let history = useHistory();
```

Comment:
Might be possible to bundle these into a custom hook.

Reply:
^